### DISCUSSION TITLE
Duplicated metadata in EDP

### DISCUSSION THREAD

**Discussion by MayteTDGeograma** [gopeg/issues-barriers#48] [General]
The EDP publishes the metadata of the national catalogs, in Spain harvesting is carried out of the following catalogs:

- Your Open Data (Smart Cities Open Data)
- Geo Data Portal Navarra region
- Geo Data Portal Spain (CODSII)
- IDEE (SDI Spanish)
- IDEAndalucia (SDI Andalusia)
- datos.gob.es (Open Data Spain)

The problem is that when a metadata is published in several catalogs, it is duplicated in the EDP.
For example:

![image](https://user-images.githubusercontent.com/94920015/207857862-fb5370bd-3b67-410f-8ee1-03f694852dd2.png)

**Comment by gmartirano** [comment]
This is a well known case indeed.
In Italy, in order to avoid MD duplicates in EDP, thanks to an agreement between EDP and AgID (national organisation managing both catalogues - geodata catalogue and open data catalogue - harvested by EDP), for all open geodata (whose metadata are published in both national catalogues) EDP harvests only the geodata catalogue.
Moreover, data providers of open geodata have to publish their metadata only in the geodata catalogue and then AgID provides an automatic conversion of the metadata in GeoDCAT-AP-IT (national profile of GeoDCAT) and publish them in the open data catalogue.
This solution has been presented by AgID in several INSPIRE Conferences.
It would be interesting to document how the problem is tackled in the other countries represented in the project.

**Reply by AndersOstmanNovogit** [reply]
The Swedish procedure is somewhat different. INSPIRE data has to be published on the geoportal, because that catalogue is harvested to the INSPIRE catalogue. Other geodata sets may be published either on the SE geoportal or the open data portal. All entries on the geodata catalogue are harvested to the open data catalogue by DIGG, the Swedish agency responsible for the open data portal. Sine EDP harvest from both the SE geodata catalogue and the SE open data catalogue, multiple entries appears. In addition, they are inconsistent with each other and the one harvested from the SE open data portal is the most correct one (the one converted by the SE agency DIGG, not the one converted by EDP).

**Comment by AndersOstmanNovogit** [comment]
This issue is reported in section 4.5.3 in D4.3.

@MayteTDGeograma : Are the duplicated metadata records identical? Do the score equally when running a MQA test on each one of them? When doing this exercise for Swedish duplicated metadata records, there are significant differences between the test results.

**Reply by AndersOstmanNovogit** [reply]
Now reported in D4.3 section 4.4.4

**Comment by gmartirano** [comment]
Dear @AndersOstmanNovogit, please don't forget that the EDP tests using the MQA tool check also optional DCAT-AP fields (associated with metadata quality assessment according to MQA methodology), whilst national open data portals generally do a conversion from ISO_19139 to DCAT-AP (sometimes GeoDCAT-AP, sometimes also toward national profiles of (Geo)DCAT-AP).
Furthermore, in the case of Italy, EDP harvests open geodata only from national catalogue of geodata (to avoid duplications) and performs a GeoDCAT-AP conversion different from the GeoDCAT-AP(IT) conversion made by the Italian open data portal.
Therefore the two tests very likely will show differences.

**Reply by AndersOstmanNovogit** [reply]
Hi @gmartirano . I have currently written the following in D4.3

There are also indications that different conversions of metadata from ISO 19115 standard to DCAT-AP produce different results. Although not investigated deeply in this study, it seems that the conversion made by EDP when harvesting from geoportals, produce results of less quality.

The reason for the last sentence is that, according to the very few tests I have made, metadata records converted by the Swedish agency DIGG scores higher on the MQA test as compared to the conversions made by EDP. But I have asked Mayte to confirm this for Spain.
